src/resources/index.html

```diff
@@ -0,0 +1,169 @@
+
+          <code class="code">
+            npm i @srcery-colors/srcery-palette
+          </code>
+          <ul class="">
+            <li class="mb-2 flex items-center">
+              <div class="list-dot"></div>
+              <a target="_blank" class="link flex items-center" href="https://github.com/srcery-colors/srcery-palette">
+                <div class="svg mr-2 w-5" x-html="icons.github"></div>
+                Github repository
+              </a>
+            </li>
+            <li class="flex items-center">
+              <div class="list-dot"></div>
+              <a target="_blank" class="flex items-center link" href="https://www.npmjs.com/package/@srcery-colors/srcery-palette">
+                <div class="svg mr-2 w-8" x-html="icons.npm"></div>
+                <div>
+                  <span class="mr-1">npm package</span>
+                  <span x-show="version">(<strong x-show="version" x-text="version"></strong>)</span>
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="mb-8">
+          <h2 class="text-xl mb-4 text-center sm:text-left uppercase">Web assets</h2>
+          <p class="mb-4">Aside from the palette itself, here are some useful web assets</p>
+          <ul class="list-disc list-inside">
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/tailwind">
+                Tailwind config
+              </a>
+            </li>
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/tridactyl">Tridactyl theme</a>
+              <span class="text-sm text-white">- This will be moved to a new repo soon</span>
+            </li>
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/userstyles">User styles</a>
+              <span class="text-sm text-white">- Using stylish extension for example</span>
+            </li>
+
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/palette">Palette</a>
+              <span class="text-sm text-white">this is deprected for the palette above, kept for compatibility</span>
+            </li>
+          </ul>
+        </div>
+        <div class="mb-8">
+          <h2 class="text-xl mb-4 text-center sm:text-left uppercase">Graphic assets</h2>
+          <p class="mb-4">
+            We keep graphic assets in a separate repo so not to inflate the
+            size of each theme. We also keep the source svgs for various
+            assets, kept in the <code class="inline-code">src</code> folder.
+          </p>
+          <p class="mb-4">
+            If you are looking for icons, logo, screenshots etc this is the place.
```

Comment:
```suggestion
            If you are looking for icons, logo, screenshots, etc., this is the place.
```

Reply:
Both period and comma in the suggestion

Reply:
Yeah.  I don't know what's correct with `etc.` in a list.

Reply:
I'll use the long form, `etcetera`, to avoid punctuation troubles. ce2ac71